File: app/src/main/java/com/theroyalsoft/telefarmer/utils/IceCandidateSocket.java
package com.theroyalsoft.telefarmer.utils;

/**
 * Created by Pritom Dutta on 5/8/23.
 */
public class IceCandidateSocket {
    private String sdp;
    private int sdpMLineIndex;
    private String sdpMid;//sdpMid


    public IceCandidateSocket(String sdp, int sdpMLineIndex, String sdpMid) {
        this.sdp = sdp;
        this.sdpMLineIndex = sdpMLineIndex;
        this.sdpMid = sdpMid;

    }

    public String getSDP() {
        return NullRemoveUtil.getNotNull(sdp);
    }

    public int getSDPMLineIndex() {
        return sdpMLineIndex;
    }

    public String getSDPMid() {
        return NullRemoveUtil.getNotNull(sdpMid);
    }

}

File: model/src/main/java/bio/medico/patient/model/socket/IceCandidateSocket.kt
package bio.medico.patient.model.socket

import bio.medico.patient.model.NullRemoveUtil

data class IceCandidateSocket(
    private val sdp: String,
    val sDPMLineIndex: Int, //sdpMid
    private val sdpMid: String
) {
    val sDP: String
        get() = NullRemoveUtil.getNotNull(sdp)
    val sDPMid: String
        get() = NullRemoveUtil.getNotNull(sdpMid)
}